radio/src/gui/212x64/splash.cpp

```diff
@@ -31,7 +31,7 @@ struct {
   const uint8_t __magic_suffix[4] = { 'S','P','E',0 };
 } splash_struct;
 static_assert(sizeof(splash_struct.splashdata) <= MAXIMUM_SPLASH_IMAGE_SIZE, "");
-const unsigned char * const splash_lbm = splash_struct.splashdata+4;
```

Comment:
The first 4 bytes do not represent the dimensions anymore?

Reply:
It is only two bytes for the dimensions and that is handled in the RleBitmap class. The +4 was for the magic, but with the strict it is not needed.